File: smartwatch/src/main/java/com/ehts/smartwatch/MainActivity.java
package com.ehts.smartwatch;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.auth.FirebaseUser;

public class MainActivity extends AppCompatActivity {

    FirebaseAuth auth;
    Button logoutbtn;
    TextView textView;
    //Button gotoHr;
    FirebaseUser user;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //initialize
        auth = FirebaseAuth.getInstance();
        logoutbtn = findViewById(R.id.logoutbtn);
        textView= findViewById(R.id.userdetails);
       // gotoHr = findViewById(R.id.gotoHr);

        user = auth.getCurrentUser();

        if(user == null){
            Intent intent = new Intent(getApplicationContext(), loginwatch.class);
            startActivity(intent);
            finish();
        }
        else{
            textView.setText(user.getEmail());
        }


   /*
        gotoHr.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(), HeartRate_Monitor.class);
                startActivity(intent);
            }
        });

 */
        logoutbtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                FirebaseAuth.getInstance().signOut();
                Intent intent = new Intent(getApplicationContext(), loginwatch.class);
                startActivity(intent);
                finish();
            }
        });

    }

}

File: ehtswatch/src/main/java/com/ehts/ehtswatch/MainActivity.kt
package com.ehts.ehtswatch

//import android.view.View
//import androidx.appcompat.app.AppCompatActivity
import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import com.google.firebase.auth.FirebaseAuth
import com.google.firebase.auth.FirebaseUser
import com.google.firebase.database.FirebaseDatabase


class MainActivity : Activity() {

        private var auth: FirebaseAuth? = null
        private var logoutbtn: Button? = null

        private var hrbtn: Button? = null
        private var textView: TextView? = null
        private var user: FirebaseUser? = null

        override fun onCreate(savedInstanceState: Bundle?) {
            super.onCreate(savedInstanceState)
            setContentView(R.layout.activity_main)

            // Initialize Firebase Authentication
            auth = FirebaseAuth.getInstance()
            hrbtn = findViewById(R.id.gotoHr)
            logoutbtn = findViewById(R.id.logoutbtn)
            textView = findViewById(R.id.userdetails)
            user = auth?.currentUser

            if (user == null) {
                val intent = Intent(applicationContext, loginwatch::class.java)
                startActivity(intent)
                finish()
            } else {
                textView?.text = user?.email
            }
            hrbtn?.setOnClickListener {

                val intent = Intent(applicationContext, EmployeesHome::class.java)
                startActivity(intent)
                finish()
            }

            logoutbtn?.setOnClickListener {
                FirebaseAuth.getInstance().signOut()
                val intent = Intent(applicationContext, loginwatch::class.java)
                startActivity(intent)
                finish()
            }
        }

}